scripts/process_data.py

```diff
@@ -28,6 +29,30 @@
 CONSOLE = Console(width=120)
 
 
+def find_tool_feature_matcher_combination(sfm_tool, feature_type, matcher_type):
+    """Find a valid combination of sfm tool, feature type, and matcher type.
+    Basically, replace the default parameters 'any' by usable value"""
```

Comment:
The `Args` and `Returns` should be documented.

Reply:
done